conn.go

```diff
@@ -383,11 +407,10 @@ func (conn *Conn) Object(dest string, path ObjectPath) BusObject {
 	return &Object{conn, dest, path}
 }
 
-func (conn *Conn) sendMessage(msg *Message) {
```

Comment:
Why is this refactor necessary? If we keep it, it should be done in a separate commit as it is unrelated to adding the interceptors.

Reply:
I wanted to have one method for all outgoing messages, done as a separate commit